walkthroughs/howto-grpc/deploy.sh

```diff
@@ -119,10 +119,29 @@ delete_mesh() {
     aws appmesh-preview delete-mesh --mesh-name $mesh_name
 }
 
+delete_images() {
+    for app in color_client color_server; do
+        echo "deleting repository..."
+        aws ecr delete-repository \
+           --repository-name $PROJECT_NAME/$app \
+           --force
+    done
+}
+
 delete_stacks() {
+    echo "delete app..."
```

Comment:
I removed these echos from here and moved more descriptive echos into the functions themselves